Resources/Storage/StorageElement.py

```diff
@@ -253,7 +254,7 @@ def getStatus( self ):
       - DiskCacheTB: float (-1 if not defined)
     """
 
-    self.log.verbose( "StorageElement.getStatus : determining status of %s." % self.name )
+    self.log.getSubLogger( 'getStatus' ).verbose( "determining status of %s." % self.name )
```

Comment:
This kind of changes - is it just a beautification of the output or there are more reasons for that ?